minvws/nl-kat-coordination
# Exclude OOIs creation from the OOI add form by OOI-types

### Changes
Exclude manual creation of OOI's with certain types. 

### Issue link

Closes ...

### Demo
These types are excluded from the ooi add form:
RESTAPI, Incident, ImageMetadata, Report, ReportData

<img width="1117" alt="image" src="https://github.com/user-attachments/assets/cadfb7e6-71e1-4631-bb9c-b40dcac4f806">



### QA notes
Check if types are excluded and cannot manipulate ooi creation from URL at ooi add form
RESTAPI, Incident, ImageMetadata, Report, ReportData

<img width="1111" alt="image" src="https://github.com/user-attachments/assets/62eed768-4d9b-4de9-aa47-1fcaeecc353e">



---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
